packages/cli/src/commands/project/list.ts

```diff
@@ -48,39 +51,69 @@ export default async function list(
   const elapsed = ms(Date.now() - start);
 
   output.log(
-    `${list.length > 0 ? 'Projects' : 'No projects'} found under ${chalk.bold(
-      contextName
-    )} ${chalk.gray(`[${elapsed}]`)}`
+    `${
+      projectList.length > 0 ? 'Projects' : 'No projects'
+    } found under ${chalk.bold(contextName)} ${chalk.gray(`[${elapsed}]`)}`
   );
 
-  if (list.length > 0) {
-    const cur = Date.now();
-    const header = [['', 'name', 'updated'].map(title => chalk.dim(title))];
-
-    const out = table(
-      header.concat(
-        list.map(secret => [
-          '',
-          chalk.bold(secret.name),
-          chalk.gray(`${ms(cur - secret.updatedAt)} ago`),
-        ])
-      ),
+  if (projectList.length > 0) {
+    const tablePrint = table(
+      [
+        [
+          'Project Name',
+          inspect ? 'Inspect Project' : 'Latest Production URL',
```

Comment:
**suggestion (non-blocking):** 

```suggestion
          inspect ? 'Inspect Lated Production Deployment' : 'Latest Production URL',
```

I think our jargon aligns better with `Inspect Deployment` instead of `Inspect Project`. What do you think of this? I couldn't think of something shorter that still made it clear.

Reply:
But `--inspect` takes you to the homepage for the Project on Vercel's dashboard, so I don't think it's correct to say "Inspect Latest Production Deployment". Would you be in favor of something along the lines of "Manage Project"?

Reply:
Ohh, I missed that part. Yeah, maybe `Inspect` is a loaded term. Or maybe I'm reading too much into it.

I do like `Manage Project` or `View Project` or just `Project`, but then the flag should probably be called something else, too.